student.rb

```diff
@@ -3,8 +3,8 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(classroom, name, age, parent_permission: true)
-    super(name, age, parent_permission: parent_permission)
+  def initialize(classroom, age, name, parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
     @classroom = classroom
   end
 
```

Comment:
- [ ] The Student class is missing the other end of the association with the classroom class. Kindly add it.

  ```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```